### ISSUE TITLE
$gel-type-touch-class confusing name

### ISSUE THREAD

**Issue by howieweiner** [bbc/gel-typography#46]
$gel-type-touch-class requires the class name of the NO touch class. Somewhat confusing

**Comment by shaunbent** [comment]
Would you prefer `$gel-type-no-touch-class` or `$gel-type-no-touch-class-name`?!?

I guess because I know what it does I never thought it might be a bit confusing.

**Comment by howieweiner** [comment]
$gel-type-no-touch-class is fine, but don't want this to become a breaking change.. unless you extend your logic to look for one or the other variable name?

**Comment by shaunbent** [comment]
Yeah I was thinking of allowing both variables and deprecating the existing one and throwing a warning if its used to suggest people upgrade to the new one.

**Comment by howieweiner** [comment]
👍 for the warning

**Comment by itsaljones** [comment]
Sounds good to me!

**Comment by randombitsUK** [comment]
This issue was raised 9 years ago, and given how little friction the existing implementation will be closed.
